app/controllers/requests_controller.rb

```diff
@@ -47,8 +46,4 @@ def set_requester_on(request)
     request.requester.name = current_user.name
     request.requester.email = current_user.email
   end
-
-  def persistence_to_zendesk_necessary?
```

Comment:
Just a comment rather than an issue - we'll probably have to revisit this logic when we remove the zendesk dependency from anonymous feedback requests.